### PR TITLE
[Review] Followups after #552 review

### PR DESCRIPTION
**Pull request by Olshansk**
## Summary

Minor follows-ups (improvements & capturing TODOs) while reviewing #552 after it was merged.

Nothing is critical, but capturing work and improvements while it's fresh.

## Issue

- #552

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [ ] Bug fix
- [x] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- [ ] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR.
    - **THIS IS VERY EXPENSIVE**, so only do it after all the reviews are complete.
    - Optionally run `make trigger_ci` if you want to re-trigger tests without any code changes
    - If tests fail, try re-running failed tests only using the GitHub UI as shown [here](https://github.com/pokt-network/poktroll/assets/1892194/607984e9-0615-4569-9452-4c730190c1d2)

## Sanity Checklist

- [ ] I have tested my changes using the available tooling
- [ ] I have commented my code
- [ ] I have performed a self-review of my own code; both comments & source code
- [ ] I create and reference any new tickets, if applicable
- [ ] I have left TODOs throughout the codebase, if applicable

## Summary by CodeRabbit

- **New Features**
  - Introduced a new command to quickly start a localnet setup without fixture regeneration.
  - A new feature specification for service management testing has been added.

- **Improvements**
  - Enhanced error handling and reporting for flaky tests.
  - Updated command to reflect the new functionality of updating existing services rather than adding them.
  - Improved clarity and correctness in function names and variable usage across various components.

- **Bug Fixes**
  - Corrected the error constant for service validation to ensure accurate error reporting.

- **Documentation**
  - Improved comments for clarity on compute units and relay costs, enhancing understanding of the service functionality.

### CODE REVIEW

**Comment by Olshansk** [comment]
Ccing @rBurgett for visibility on this. No pressure for review but feel free to add comments.

**Comment by Olshansk** [comment]
@adshmh Could you PTAL at this? I think some of the TODOs here are also good onboarding tasks to get really in the weeds of this repo to take on larger ticket items.

**Review by red-0ne — COMMENTED** [review]
Great comments, this blends well with the source owner feature.

Left some questions and comments but also seeing some `go-test` check failures.

File: proto/poktroll/shared/service.proto [hunk 1]
```diff
@@ -14,7 +14,9 @@ message Service {
   // TODO_BETA: Name is currently unused but acts as a reminder that an optional onchain representation of the service is necessary
   string name = 2; // (Optional) Semantic human readable name for the service
 
-  // Used alongside the global 'compute_units_to_tokens_multipler' to calculate the cost of a relay for this service
+  // The cost of a single relay for this service in terms of compute units.
+  // Must be used alongside the global 'compute_units_to_tokens_multipler' to calculate the cost of a relay for this service.
+  // cost_per_relay_for_specific_service = compute_units_per_relay_for_specific_service * compute_units_to_tokens_multipler_global_value
```
[the remark this answers is not in the document]
Comment (Olshansk) [reply proto/poktroll/shared/service.proto:19]: This is part of the Service function, so it's just a single value.

**Review by red-0ne — APPROVED** [review]
LGTM.

**Pull request merged**